helm/streampark/templates/streampark.yml

```diff
@@ -60,7 +60,6 @@ spec:
             {{- toYaml .Values.spec.container.env | nindent 12 }}
           securityContext:
             privileged: false
-          command: ["bash","-c","bash ./bin/streampark.sh start_docker"]
```

Comment:
Can you please explain why remove start_docker?

Reply:
https://github.com/apache/incubator-streampark/blob/dev-2.1.5/docker/Dockerfile#L79

Reply:
Remove command: ["bash","-c","bash ./bin/streampark.sh start_docker"] to fix the issue where it cannot start.

Reply:
Thanks for your clarification